open-craft/xblock-eoc-journal
# (BB-1359) Add translations configuration and automation

This PR adds necessary configuration files to create translations. Working with translations is automated by using make targets which are explained in the README.

# Testing instructions:

1. Extract translations: `make extract_translations`
2. Build dummy translations: `make build_dummy_translations`
3. Validate translations: `make validate_translations`
4. Run lms, go to `/update_lang/` and update language to `eo`
5. Make sure strings are translated.
